### PR TITLE
Db duplicate tables

### PR DESCRIPTION
**Pull request by JosiahRegencia**
This is for issue #20

### CODE REVIEW

File: bassist/bassist/parser/log_file/debs_stdout.py [hunk 1]
```diff
@@ -2,10 +2,11 @@
 # See the file LICENSE for copying permission.
 from . import common
 from ...systems import package
-from results import store_debs
-from run_sql import execute_sql
+
+debs = []
```
Comment (greenmoss) [inline bassist/bassist/parser/log_file/debs_stdout.py:6]: Most of the code is part of the other PR #17, which was merged. Please change this PR so it only includes the commits relevant to the DB duplicates issue.

**Review by greenmoss — CHANGES_REQUESTED** [review]

File: bassist/bassist/parser/log_file/debs_stdout.py (hunk 1 above)
Comment (greenmoss) [reply bassist/bassist/parser/log_file/debs_stdout.py:6]: screen shot 2017-08-12 at 11 47 14 am

Comment (JosiahRegencia) [reply bassist/bassist/parser/log_file/debs_stdout.py:6]: This commit should be part of the PR: Added check_exists functions in DB    2f37008

 I just added it now

Comment (greenmoss) [reply bassist/bassist/parser/log_file/debs_stdout.py:6]: Sorry I got it wrong. Here's the updated image:
screen shot 2017-08-12 at 11 47 14 am

Comment (JosiahRegencia) [reply bassist/bassist/parser/log_file/debs_stdout.py:6]: working on it

**Comment by greenmoss** [comment]
I see the fix commits, but this PR still has old history. You might need to delete this branch and try again. Try this:

- Check out master
- Create a new branch
- Merge your DB commits
- Push the branch
- Open a new PR

Then I'll close this PR as invalid.

**Comment by JosiahRegencia** [comment]
Will do. I'm sorry. I've never done this part of git before

**Comment by greenmoss** [comment]
No need to apologize; I am learning too :)

**Pull request merged**